### PR TITLE
Update docs and add state for junos_lacp

### PR DESCRIPTION
**Pull request by danielmellado**
This commit adds gathered state for junos_lacp and updates the
documentation to the new resource modules versioning for collections.

### CODE REVIEW

File: plugins/modules/junos_lacp.py [hunk 1]
```diff
@@ -30,15 +30,13 @@
...
-DOCUMENTATION = """module: junos_lacp
-short_description: Manage Global Link Aggregation Control Protocol (LACP) on Juniper
-  Junos devices
+ANSIBLE_METADATA = {"metadata_version": "1.1", "supported_by": "network"}
+
+DOCUMENTATION = """
+---
+module: junos_lacp
+version_added: "1.0.0"
+short_description: Global Link Aggregation Control Protocol (LACP) Junos resource module.
```
Comment (samccann) [inline plugins/modules/junos_lacp.py:39]:
```suggestion
short_description: Global Link Aggregation Control Protocol (LACP) Junos resource module
```

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by pabelanger** [comment]
Until we land job redesign, I am going to close this issue (and reopen) to free up some capacity.

**Pull request merged**